package/MDAnalysis/exceptions.py

```diff
@@ -35,6 +35,26 @@ class SelectionError(Exception):
 class NoDataError(ValueError, AttributeError):
     """Raised when empty input is not allowed or required data are missing.
 
+    This exception is used in various scenarios:
```

Comment:
Part of the issue this is trying to fix is that the docstring is too vague and unspecific. `Various` is still quite unspecific -- I'd suggest using stricter language to make it clear what this exception is intended to address.

```suggestion
    This exception is raised in the following scenarios:
```

Reply:
Sure